src/configuration/routes/https.md

```diff
@@ -129,3 +131,30 @@ tls:
             ### Several lines of different random characters here ###
             -----END CERTIFICATE-----
 ```
+
+## Automatic certificate renewal
+
+It is possible to set a variable from a cron task, which in turn will cause the site to redeploy.  If the Let's Encrypt certificate is due to expire in less than one month it will automatically renew at that time.  That makes it feasible to set up auto-renewal of the Let's Encrypt certificate.  The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.
+
+You will first need to install the CLI in your application container.  See the section on [API tokens](/gettingstarted/cli/api-tokens.md) for instructions on how to do so.
+
+Once the CLI is installed and an API token configured you can add a cron task to run once a month and set a placeholder variable.  For example:
+
+```yaml
+crons:
+    renewcert:
+        # Force a redeploy at 10 am (UTC) on the 3rd of every month.
+        spec: '0 10 3 * *'
+        cmd: |
+            if [ "$PLATFORM_BRANCH" = master ]; then
+                platform variable:set _redeploy "$(date)" --no-wait
```

Comment:
Use `--yes` to turn off interactivity (including update checks, and any confirm steps that the command may ask)